app/assets/javascripts/initializers/initializeBaseUserData.js

```diff
@@ -103,28 +103,18 @@ function addRelevantButtonsToComments(user) {
   }
 }
 
-function initializeBaseUserData() {
-  const user = userData();
-  const userProfileLinkHTML =
-    '<a href="/' +
-    user.username +
-    '" id="first-nav-link" class="crayons-link crayons-link--block"><div>' +
-    '<span class="fw-medium block">' +
-    user.name +
-    '</span>' +
-    '<small class="fs-s color-base-50">@' +
-    user.username +
-    '</small>' +
-    '</div></a>';
-  document.getElementById(
-    'user-profile-link-placeholder',
-  ).innerHTML = userProfileLinkHTML;
```

Comment:
Is this `.innerHTML = userProfileLinkHTML` not needed anymore?

Reply:
Line 108-111 is doing the same thing, so this was a duplicate functionallity.

Reply:
Ohh I see 👍